app/use_cases/meteors/meteors_service.js

```diff
@@ -0,0 +1,20 @@
+const mapAsteroids = require("./meteors_transformer");
+const {fetchAsteroids} = require("../../repository/nasa_api");
+
+async function getMeteors(startDate, endDate, isDangerous, count) {
+    try {
+        const asteroids = await fetchAsteroids(startDate, endDate);
+        return mapAsteroids(asteroids, filterPredicate(isDangerous), count);
+    } catch (error) {
+        console.error('Ups, something went wrong: ', error);
+        throw error;
```

Comment:
create please custom Exception with possibility to provide statusCode and errorMessage

Reply:
Added error handling, thank you for paying my attention to it☺